validator/accounts/account.go

```diff
@@ -118,7 +118,7 @@ func NewValidatorAccount(directory string, password string) error {
 }
 
 // Exists checks if a validator account at a given keystore path exists.
-func Exists(keystorePath string) (bool, error) {
+func Exists(keystorePath string, assertNonEmpty bool) (bool, error) {
```

Comment:
Since `Exists` is exported. Can we please add a comment on what `assertNonEmpty` is?